assets/javascript/app.js

```diff
@@ -0,0 +1,175 @@
+// array to store trivia questions
+var questions = [
+  {
+    question: "What house at Hogwarts does Harry belong to?",
+    A: "Gryffindor",
+    B: "Ravenclaw",
+    C: "Slytherin",
+    D: "Hufflepuff",
+    correctAsw: "A"
+  },
+  {
+    question: "What position does Harry play on his Quidditch team?",
+    A: "Bludger",
+    B: "Seeker",
+    C: "Chaser",
+    D: "Keeper",
+    correctAsw: "B"
+  },
+  {
+    question: "Who is Fluffy?",
+    A: "Hagrid's dragon",
+    B: "A three-headed dog",
+    C: "Hermione's cat",
+    D: "Harry's owl",
+    correctAsw: "B"
+  }
+];
```

Comment:
Great choice to use objects to organize the questions, choices and answers! You could probably have leveraged this strategy further for your image URLS too!